packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -136,11 +137,39 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
       }
     }
 
+    if (credential.app?.categories.includes(AppCategories.crm)) {
+      const metadata = EventTypeMetaDataSchema.parse(eventType.metadata);
+      const appSlugToDelete = credential.app?.slug;
+
+      if (appSlugToDelete) {
+        const appMetadata = removeAppFromEventTypeMetadata(appSlugToDelete, metadata);
+
+        await prisma.$transaction(async () => {
```

Comment:
This code seems same as the following code for payment app handling. We should merge these two.